sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -165,8 +165,9 @@ export interface QueueOptions {
   duplicateDetectionHistoryTimeWindow?: string;
 
   /**
-   * The URL of Service Bus queue to forward deadlettered messages to.
-   *
+   * Relative path of the messaging entity the dead-lettered messages are to be forwarded to.
+   * For e.g., `samplequeue` for configuring a sample queue as target, or
+   * `autoforwardingtopic/samplesubscription` for a sample subscription.
```

Comment:
When using queues, can the dead lettered messages be forwarded to a subscription and vice versa?

Reply:
Yes, just updated the PR. The newer documentation and tests should give better idea on usage.

Looks like the `forwardDeadLetterMessagesTo` property is getting set and works as expected, but not the `forwardTo`.

Reply:
> Looks like the forwardDeadLetterMessagesTo property is getting set and works as expected, but not the forwardTo.

Is that the current state of the PR or the state of the PR before your fixes in commits https://github.com/Azure/azure-sdk-for-js/pull/6382/commits/51ec679dbe4d6ab025a13aaebe6113e9bed0be90 and https://github.com/Azure/azure-sdk-for-js/pull/6382/commits/1a316f39ab2567d241350e29a72ef6fcaad21129 make the same set of changes to both `forwardTo` and `forwardDeadLetterMessagesTo` properties